### PR TITLE
{bio}[intel/2019.01] TM-align v20180426

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@pescobar As with the other version, could you submit a test report for this?
I can't currently test as I don't have this version of the compiler installed.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dcdd07757a3545ff9a52b5684b348071 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/eb143daa39b1f6431be17809982c912d for a full test report.

**Comment by verdurin** [comment]
@pescobar could you rename the downloaded source file, as in:
https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources

Please submit an update for the `foss` version that addresses this too.

Also, does it really need to be using the `intel` toolchain?

**Comment by pescobar** [comment]
I have updated the intel and foss easyconfigs to rename the downloaded tarball

**Comment by pescobar** [comment]
@verdurin using intel toolchain is not mandatory. I just added the easyconfig to double check that the custom buildopts were working fine with different toolchains

File: easybuild/easyconfigs/t/TM-align/TM-align-20180426-intel-2019.01.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+name = 'TM-align'
+version = '20180426'
+
+homepage = 'https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/'
+description = """This package unifies protein structure alignment and RNA structure alignment
+ into the standard TM-align program for single chain structure alignment, MM-align program for
+ multi-chain structure alignment, and TM-score program for sequence dependent structure superposition."""
+
+toolchain = {'name': 'intel', 'version': '2019.01'}
```
Comment (smoors) [inline easybuild/easyconfigs/t/TM-align/TM-align-20180426-intel-2019.01.eb]: @pescobar any specific reason why you're not using the `2019a` intel version? we should try to stick with the default toolchains if possible.

Comment (pescobar) [reply easybuild/easyconfigs/t/TM-align/TM-align-20180426-intel-2019.01.eb]: the only reason is that I have no `2019a` installed in my system but I will update the easyconfig

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic407 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8c36259d049169ee691a6d37168458a6 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @pescobar!

**Pull request merged**